pkg/types/validation/installconfig.go

```diff
@@ -295,12 +295,16 @@ func validateNetworkingIPVersion(n *types.Networking, p *types.Platform) field.E
 	return allErrs
 }
 
-func validateNetworking(n *types.Networking, fldPath *field.Path) field.ErrorList {
+func validateNetworking(n *types.Networking, singleNodeOpenShift bool, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if n.NetworkType == "" {
 		allErrs = append(allErrs, field.Required(fldPath.Child("networkType"), "network provider type required"))
 	}
 
+	if singleNodeOpenShift && n.NetworkType == string(operv1.NetworkTypeOpenShiftSDN) {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "only NetworkType OVNKubernetes is currently supported on Single Node OpenShift"))
```

Comment:
this is not accurate. CUs do inject cillium, callico .... I dont think we claim it's unsupported. 
I would rather say that SDN is not, and the default is OVN. Or somehting in between those lines.

Reply:
This the message we present in assisted: https://github.com/openshift/assisted-service/pull/3419/files#diff-a712b20e2e375b0286d437906cfa73f4423a506449c19e71ba14633cd89dd01bR270
but i think stating explicitly about ovn as a default is better

Reply:
Ok. Thanks for pointing out that other networkTypes are supported. In this validation check, I have reworded the error to say that OpenShiftSDN is invalid but don't think it is the right place to say what the default value is. There are other sections that have already been updated to indicate the change in default.